src/display/annotation_layer.js

```diff
@@ -615,6 +615,7 @@ class AnnotationElement {
         borderStyle: 0,
         id: `popup_${data.id}`,
         rotation: data.rotation,
+        noRotate: data.noRotate,
```

Comment:
I think we should always set `noRotate` to `true` because as far as I can tell it doesn't really make sense to have to break your neck in order to try to read a rotated popup.
@timvandermeij wdyt ?

Reply:
As far as I know, popup aren't printed and if they're because they've an appearance stream we don't care about the rotation angle in the viewer itself. Am I missing something ?